examples/internal/proto/examplepb/a_bit_of_everything.swagger.json

```diff
@@ -2670,12 +2872,26 @@
             "pattern": "publishers/[^/]+/books/[^/]+"
           },
           {
-            "name": "body",
-            "description": "The book to update.\n\nThe book's `name` field is used to identify the book to be updated.\nFormat: publishers/{publisher}/books/{book}",
+            "name": "book",
+            "description": "An example resource type from AIP-123 used to test the behavior described in\nthe CreateBookRequest message.\n\nSee: https://google.aip.dev/123",
```

Comment:
I think this is using the wrong description - this is the comment above the `Book` message, but it should probably be the comment above the `UpdateBook.Book` field (as it was before).

Reply:
Working on this (see above)

Reply:
Done.